build.proj

```diff
@@ -1,6 +1,7 @@
 <?xml version="1.0" encoding="utf-8"?>
 <Project ToolsVersion="12.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
   <Import Project="dir.props" />
+  <Import Project="$(ToolsDir)\$(BuildToolsPackageName).$(BuildToolsVersion)\lib\packages.targets" />
```

Comment:
I suspect ToolsDir isn't the same as it is in CoreFX repo which is hwy you are using this. Perhaps we should still capture these type of paths in the dir.props file so they can be more easily shared.